Iterate/SDK/Iterate.swift

```diff
@@ -21,10 +21,13 @@ public final class Iterate {
     public static let PreviewParameter = "iterate_preview"
     
     // Current version number, will be updated on each release
-    private static let Version = "0.1.1"
+    static let Version = "0.1.1"
```

Comment:
Possible you want to bump this?

Reply:
Not yet, going to wait until all of the P1 fixes are done until packaging up another release (not every commit need a version)